src/codegen/properties/attribute.py

```diff
@@ -22,9 +22,10 @@ def __init__(self, id, attribute_metadata):
         self._python_class_name = attribute_metadata["python_class_name"]
         self._handle_parameters = []
         self._object_constructor_params = []
-        if "handle_parameters" in attribute_metadata:
-            for name, parameter_data in attribute_metadata["handle_parameters"].items():
-                self._handle_parameters.append(Parameter(name, parameter_data))
+        self._handle_parameters = [
+        Parameter(name, parameter_data)
+        for name, parameter_data in attribute_metadata.get("handle_parameters", []).items()
+        ]
```

Comment:
This formatting looks a bit off. I don't think we have [black](https://pypi.org/project/black/) integrated into the build here, but it would probably make something prettier.